web/src/components/legend/LegendContainer.tsx

```diff
@@ -11,7 +11,7 @@ export default function LegendContainer(): ReactElement {
   const isWindLayerEnabled = useAtomValue(isWindLayerEnabledAtom);
 
   return (
-    <div className="invisible  flex w-[224px] flex-col rounded bg-white/90 px-1 py-2 shadow-xl backdrop-blur-sm sm:visible dark:bg-gray-800">
+    <div className="flex min-w-64 max-w-min flex-col gap-2 rounded-2xl bg-white/80 p-2 backdrop-blur dark:bg-gray-800/80">
```

Comment:
I think it is a bit hard to see the difference between this element and the background in light mode now that there is no shadow.

Before:
![image](https://github.com/user-attachments/assets/e97407c3-f956-46a4-a427-66d170fccbf6)

After:
![Screenshot 2024-07-26 at 09 05 10](https://github.com/user-attachments/assets/769920bb-e76c-4fb8-9503-d5881e68de3f)

Reply:
I'll double check how we do it for the other translucent elements. 👍🏼